storage/tianmu/system/cacheable_item.cpp

```diff
@@ -23,153 +23,172 @@
 
 namespace Tianmu {
 namespace system {
-CacheableItem::CacheableItem(char const *owner_name, char const *object_id, int _default_block_size) {
-  default_block_size = _default_block_size;
+CacheableItem::CacheableItem(char const *owner_name, char const *object_id, int default_block_size) {
+  default_block_size_ = default_block_size;
+  max_file_id_ = 0;
+  max_file_pos_ = 0;
+  no_block_ = 0;
+  cur_file_number_ = -1;
+  file_name_ = nullptr;
   DEBUG_ASSERT(owner_name != nullptr);
   DEBUG_ASSERT(object_id != nullptr);
-  // copy the temporary folder first
-  filename = nullptr;
-
-  {
-    // read the configuration parameter
-    std::string temp_filename = tianmu_sysvar_cachefolder;
-    filename_n_position = temp_filename.length();
-    filename = new char[filename_n_position + 37];  // "...path.../XXXXXXnnnnnnAAAAAAAABBBBBBBB.tianmu_tmp"
-    std::strcpy(filename, temp_filename.c_str());
-    if (filename[filename_n_position - 1] != '/' && filename[filename_n_position - 1] != '\\') {
-      filename[filename_n_position] = '/';
-      filename_n_position++;
-    }
-    filename_n_position += 6;
-  }
 
-  if (filename == nullptr) {
-    // if the temporary path is not set, use the current folder
-    filename = new char[36];  // "XXXXXXnnnnnnAAAAAAAABBBBBBBB.tianmu_tmp"
-    filename_n_position = 6;
+  constexpr size_t MIN_OWN_NAME_LEN = 3;
```

Comment:
The `constant names` should follow:
https://google.github.io/styleguide/cppguide.html#Constant_Names